templates/cicd/pipeline_cfn.yml

```diff
@@ -291,8 +280,19 @@ Resources:
                 RoleArn: arn:aws:iam::{{$stage.AccountID}}:role/{{$.AppName}}-{{$stage.Name}}-CFNExecutionRole
               InputArtifacts:
                 - Name: BuildOutput
-              RunOrder: 2
+              RunOrder: 1
```

Comment:
Can we keep this as RunOrder 2, so that the deployments happen after the approval button

Reply:
Done!